src/store/slices/users/authSlice.ts

```diff
@@ -1,16 +1,18 @@
-import { signInUser } from './authService';
+import { signInUser, signInWithGoogle } from './authService';
 import { createSlice } from '@reduxjs/toolkit';
 import { PayloadAction } from '@reduxjs/toolkit';
+import { RegisterUserData } from '../../../pages/register/Register';
 
 export interface TUser {
-  displayName: string | null;
   email: string | null;
   emailVerified: boolean;
-  isAnonymous: boolean;
-  phoneNumber: string | null;
-  photoURL: string | null;
+  phoneNumber?: string | null;
```

Comment:
```suggestion
  phoneNumber?: string;
```

Reply:
I cant remove that null becuase that is the default type of these values that come from userCredentials (firebase).
They are default as string | null from firebase.

Reply:
Atunci nu are sens sa pui semnul intrebarii